cfme/common/provider.py

```diff
@@ -709,6 +709,11 @@ def refresh_provider_relationships(self, from_list_view=False):
         except IndexError:
             raise Exception("Provider collection empty")
 
+    def refresh_provider_relationships_and_wait(self, wait=600, delay=1, refresh_delta=10):
```

Comment:
good enhancement :+1:. but I think there is no need in one more method in this case. 
Could you consider just passing wait parameter to refresh_provider_relationships instead ?